templates/admin.php

```diff
@@ -5,7 +5,7 @@
 /** @var array $_ */
 ?>
 <form id="user-saml" class="section" action="#" method="post">
-	<h2><?php p($l->t('SAML')); ?></h2>
+	<h2 class="inlineblock"><?php p($l->t('SAML')); ?></h2>
```

Comment:
This allows the save notification to appear on the same level right beside the heading